packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -2189,6 +2190,15 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         // Set the ovmADDRESS to the _from so that the subsequent call frame "comes from" them.
         messageContext.ovmADDRESS = _from;
 
+        // Burn gas to account for the OVM_ECDSAContractAccount `execute` code
+        // that is skipped during `eth_call`
+        uint256 i;
+        uint256 initialGas = gasleft();
+        uint256 gasToConsume = Lib_IntrinsicGas.ecdsaContractAccount(_transaction.data.length);
+        while(initialGas - gasleft() < gasToConsume) {
+            i++;
+        }
+
```

Comment:
How does this correctly estimate the gas cost of performing a CREATE if it does not use `Lib_ExecutionManagerWrapper.ovmCREATE(..)`? It seems to me these could be significantly divergent estimates

Reply:
Ah looked at it more--is the reason why this works is because `Lib_ExecutionManagerWrapper.ovmCREATE(..)` itself takes a `gasLimit` and **that** contract makes the call to the EM with the correct amount of gas?

If that is the case, then one notable implication is that CREATEs require more gas to complete their execution than CALLs.

Reply:
I did try using the wrapper here an consistently got an error message around the `eth_estimateGas` going over the block gas limit